src/components/icons/SVGWrapper/SVGWrapper.tsx

```diff
@@ -0,0 +1,33 @@
+import { ReactNode, forwardRef } from "react";
+import { svgWrapper, SCVWrapperVaraints } from "./SVGWrapper.css";
+
+export type SVGWrapperProps = SCVWrapperVaraints & {
+  className?: string;
+  viewBox?: string;
+  color?: string;
+  children: ReactNode;
+};
+
+export const SVGWrapper = forwardRef<SVGSVGElement, SVGWrapperProps>(
+  (
+    { className, size, viewBox = "0 0 24 24", color = "black", children },
+    ref
+  ) => {
+    return (
+      <svg
```

Comment:
**suggestion**: What about using here `<Box/>`?

I think you can use [<Box/>](https://github.com/saleor/macaw-ui/blob/canary/src/components/Box/Box.tsx) component here and pass color as a prop

Reply:
In this case I don't think so as Box doesn't support svg props as viewBox